src/styles/page.css

```diff
@@ -0,0 +1,20 @@
+@import "form.css";
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(../fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(../fonts/Avenir-Heavy.ttf);
+}
```

Comment:
```suggestion
@font-face {
  font-family: Avenir;
  font-weight: 300;
  src: url(../fonts/Avenir-Book.ttf);
}

@font-face {
  font-family: Avenir;
  font-weight: 600;
  src: url(../fonts/Avenir-Heavy.ttf);
}
```

Reply:
you're overriding first font with the second one. What's the point?